embed/src/app.py

```diff
@@ -171,62 +161,24 @@ def embed_sentences(model: str, sentences: list[str]):
     return output
 
 
-def poll_queue_every(queue_uri: str, queue_name: str, interval: int = 5):
```

Comment:
ive tested this IRL, there is no need to poll as pika handles this via the callback mechanism